apps/site/next.dynamic.constants.mjs

```diff
@@ -19,6 +20,8 @@ export const IGNORED_ROUTES = [
     locale !== defaultLocale.code && /^blog/.test(pathname),
   // This is used to ignore all pathnames that are empty
   ({ locale, pathname }) => locale.length && !pathname.length,
+  // This is used to ignore download routes for Node.js versions and downloads archive page
+  ({ pathname }) => /^download\/(v\d+(\.\d+)*|archive)$/.test(pathname),
```

Comment:
Do we need this ignore? btw instead of the URLs being `/download/vXX/archive` it should be `/download/archive/{xVV}` this would probably remove the need to also have this ignore.

Reply:
Is this `/download/v{XX}` and `/download/archive`?

Reply:
I feel the `/download/archive` url should be more of a redirect to whatever is the latest version.

Reply:
It'd only make sense if we had a `/download/archive` and `/download/archive/v{XX}`

Reply:
Updated to reference the current version. Example preview URLs;

- https://nodejs-fkzg2ar1a-openjs.vercel.app/en/download/archive
- https://nodejs-fkzg2ar1a-openjs.vercel.app/en/download/archive/v17.9.1